docs/level_0/first_example_component.md

```diff
@@ -0,0 +1,136 @@
+
+Line (4) contains a reference to the helm chart for the nginx. The helm chart is stored in an OCI registry because 
+Landscaper is currently not able to fetch helm charts from helm chart repositories.
+
+Line (5) and (6) define the name of the nginx helm installation, and the namespace into which it will be deployed. In a 
+real world scenario it would be a good idea to provide the value of the namespace as import data to get a better 
+reusable component.
+
+You find the DeployItem in file [deploy-execution-nginx.yaml](./resources/blueprint/deploy-execution-nginx.yaml).
+
+## Blueprint
+
+Next, we specify the Blueprint. The Blueprint consists of a [directory](./resources/blueprint) with the following 
+structure:
+
+```
+└── blueprint
+    ├── blueprint.yaml
+    └── deploy-execution-nginx.yaml
+```
+
+The Blueprint contains a
+[blueprint.yaml](./resources/blueprint/blueprint.yaml) with the following content, and the 
+[deploy-execution-nginx.yaml](./resources/blueprint/deploy-execution-nginx.yaml) of the DeployItem. 
+ 
+```yaml
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+
+imports:                                                   # (1)
+- name: target-cluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster 
+
+deployExecutions:                                          # (2)
+- name: nginx-execution 
+  file: /deploy-execution-nginx.yaml
+  type: GoTemplate
+```
+
+The `imports` section (1) defines the interface of the Blueprint. In this example, there is only one import parameter 
+with name `target-cluster`. This parameter will be used to provide the access data to the target cluster where the 
+nginx should be deployed to. The `target-cluster` parameter has a predefined type 
+`landscaper.gardener.cloud/kubernetes-cluster`. A value of this type is the name of a custom resource `Target` 
+containing the access data for a kubernetes cluster.
+
+By means of the import parameter `target-cluster` the Blueprint becomes reusable, and you can deploy an nginx on different 
+clusters.
+
+The `deployExecutions` section (2) contains a reference to the file of the DeployItem that we have created in the 
+previous step. 
+
+We have uploaded the Blueprint into an OCI registry. You find the Blueprint OCI artifact 
+[here](https://eu.gcr.io/sap-gcp-cp-k8s-stable-hub/examples/landscaper/docs/blueprints/github.com/gardener/landscaper/first-example).
+You can upload a Blueprint by yourself to another OCI registry with the help of the Landscaper CLI command 
+[landscaper-cli blueprints push](https://github.com/gardener/landscapercli/blob/master/docs/reference/landscaper-cli_blueprints_push.md).
+
+## Component Descriptor
+
+Our component needs a Component Descriptor that contains the list of all required resources and how to access them.
+Here, these are the Blueprint, and the helm chart of the nginx with their corresponding OCI references.
```

Comment:
we have never explained where the helm chart if coming from and how it is uploaded?